### DISCUSSION TITLE
Disable displaying log

### DISCUSSION THREAD

**Discussion by fredattack** [spatie/ray#636] [Q&A]
Hi, 

Thanks for you unbelievable job.. you make my life easier !! Proud to be Belgian... ;-) 

Is the re any way to disable displaying the logs ? maybe by log level? 
            Log :: debug ( 'hydrateShipment ' , [ 'data' => $data ] );
Kind regards.

**Comment by freekmurze** [comment]
In the config file you can disable that behaviour. Take a look in the docs 👍
